### ISSUE TITLE
COMMUNICATION_ERROR device offline (9009) l930 and p100

### ISSUE THREAD

**Issue by kylelliott** [wildcs/oh3_tapocontrol#12]
![image](https://github.com/wildcs/oh3_compiled-addons/assets/23058718/00099783-70ab-4124-8499-47d632d6df8d)
have tp link changed something again

**Comment by wildcs** [comment]
Can you ping your device? May IP-Address changed by your router?

**Comment by kylelliott** [comment]
@wildcs yes have checked ip address and they are correct this for l930 and p100

**Comment by wildcs** [comment]
I uploaded a new snapshot for 4.2.0 with some fixes. Maybe this may help. 
version for 4.1.0 stable will follow next days.
Happy new year

If not i need a tracelog. See oh-documentation how to enable it.

**Comment by kylelliott** [comment]
Thanks no worries not planning to update as i always end breaking other stuff so i have to wait for 4.1.0 version

**Comment by kylelliott** [comment]
@wildcs has 4.1.0 been updated on github yet as cant see it

**Comment by kylelliott** [comment]
@wildcs anynew on this

**Comment by wildcs** [comment]
Sorry but i already spent weeks (calculated to 24/7) of work and bought some hardware for this project in my freetime. 
I also have normal work, a family with kids and other projects i have to manage.
So you have to wait until i have enough time to develop different versions. 
If you will help you can try a test with snapshot 4.2. or minimaly post a tracelog as i said sbove.

**Comment by kylelliott** [comment]
HI

Affter trying all the plugiing still getting same error

**Comment by wildcs** [comment]
Again. Without a tracelog it's like looking in an forrest.
Let me guess one more thing. Did you try to set protocol to klap? (Advanced Settings)

**Comment by kylelliott** [comment]
how do i run tracelog 

klap was already set as it working then just ramdonly stop

**Comment by kylelliott** [comment]
im running openhab on windows

**Comment by kylelliott** [comment]
![image](https://github.com/wildcs/oh3_compiled-addons/assets/23058718/83803c38-be7e-4bcf-8b0f-dcb6bd3d851f)

**Comment by kylelliott** [comment]
if want teamviewer  access to see what going on let me know as my openhab is public facing

**Comment by kylelliott** [comment]
[openhab.log](https://github.com/wildcs/oh3_compiled-addons/files/14076326/openhab.log)
[events.log](https://github.com/wildcs/oh3_compiled-addons/files/14076327/events.log)

**Comment by kylelliott** [comment]
Like loads of  people all over form struggling to get this working

**Comment by kylelliott** [comment]
@wildcs seem to work  only when you add the device manually rather than thought the cloud connector
